src/test/java/io/strimzi/kafka/metrics/KafkaMetricsCollectorTest.java

```diff
@@ -51,10 +50,15 @@ public void testCollect() {
         metrics = collector.collect();
         assertTrue(metrics.isEmpty());
 
-        // Adding a non-numeric metric does nothing
-        collector.addMetric(buildNonNumericMetric("name2", "group"));
+        // Adding a non-numeric metric converted
```

Comment:
Tests are best when they are focused on testing a single responsibility.     That makes it really clean what features are covered.    It is also more obvious what's broken when tests fail.

So if I were adding this feature, I'd be adding a new test case.. testNonNumericMetric